packages/react-context-selector/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "@fluentui/react-context-selector",
+  "version": "0.52.0",
+  "description": "React useContextSelector & useContextSelectors hooks in userland",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.1",
+    "@fluentui/react-conformance": "^1.0.0",
+    "@fluentui/scripts": "^1.0.0",
+    "@testing-library/react": "^10.4.9",
+    "@testing-library/react-hooks": "^5.0.3",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-is": "^17.0.1",
```

```suggestion
    "react-is": "^16.6.3",
```

Can we reuse there the same version as in other packages?

---

sounds good